django/conf/urls/static.py

```diff
@@ -17,7 +17,7 @@ def static(prefix, view=serve, **kwargs):
         # ... the rest of your URLconf goes here ...
     ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
     """
-    # No-op if not in debug mode or an non-local prefix
+    # No-op if not in debug mode or a non-local prefix
```

Comment:
I think this comment should have been moved above `return []` in 8ae90c58ac6f87aec711701b726fda7b261544cf.